stable/grafana/templates/deployment.yaml

```diff
@@ -13,6 +13,8 @@ spec:
   template:
     metadata:
       annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
```

Comment:
Grafana does not function without this as Grafana needs a hard redeploy in order to pick up config changes. I am checking as we speak if this is default behaviour for other charts or not.